libs/components/lookup/src/assets/locales/resources_en_US.json

```
@@ -67,6 +67,10 @@
     "_description": "Text displayed when the user has more than the maximum items selected",
     "message": "{0} items selected"
   },
+  "skyux_search_aria_label_descriptor": {
+    "_description": "Default aria label for a search when a descriptor is given",
```

```suggestion
    "_description": "Default ARIA label for a search when a descriptor is given",
```
Also, same question as before about whether the "when a descriptor is given" clause is accurate.

---

Same response as on filter.